app/controllers/contributors_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ContributorsController < ApplicationController
+  before_action :check_if_feature_flag_is_on
+
+  def new
+    @contributor = Contributor.new
+  end
+
+  def create
+    @contributor = Contributor.new(contributor_params)
+    @suggested_councillor = SuggestedCouncillor.find(params[:contributor][:suggested_councillor_id])
+    if @contributor.save
+      @suggested_councillor.contributor_id = @contributor.id
```

Comment:
Could this be done on one line with something like `@suggested_councillor.update(confributor: @contributor)`?